public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```

Comment:
Please remove anything extra which are part of template and not needed for the app.

Reply:
This is meta tag why we need to remove it?

Reply:
I don't mind keeping the meta tag but update the content something relevant to Task Management Application. 

There were lot of comments more related to create-react-app template which are not exactly needed in the application.

Reply:
I have updated in details